build.gradle

```diff
@@ -31,6 +28,18 @@ sourceSets {
 	}
 }
 
+loom {
+	accessWidenerPath = file("src/main/resources/trinkets.accesswidener")
+
+	runs {
+		testmodClient {
+			client()
+			ideConfigGenerated project.rootProject == project
```

Comment:
```suggestion
```
Little nit pick: No need to set `ideConfigGenerated` to `project.rootProject == project` as this is the default behaviour. In the root project its worth setting `ideConfigGenerated` if you want to disable it.
😜